sqlx-postgres/src/listener.rs

```diff
@@ -366,7 +369,7 @@ impl Drop for PgListener {
             };
 
             // Unregister any listeners before returning the connection to the pool.
-            crate::rt::spawn(fut);
+            crate::rt::spawn(fut.instrument(self.span.clone()));
```

Comment:
Because arguably, the user could also expect this to capture `Span::current()` or create a child span specifically for this scope.